internal/test/fixtures/item-omitted-fields/rolebinding-template.yml

```diff
@@ -0,0 +1,13 @@
+apiVersion: authorization.openshift.io/v1
+groupNames: null
+kind: RoleBinding
+metadata:
+  name: admin-0
+roleRef:
+  name: admin
+subjects:
+- kind: ServiceAccount
+  name: jenkins
+  namespace: foo-cd
+userNames:
+- system:serviceaccount:foo-cd:jenkins
```

Comment:
Should there a be or is there already a test where userNames/groupNames are:
- not in the template
- inconsistent with the subjects. This could prove they are ignored I guess

Reply:
Regarding "inconsistent with the subjects": That would need an ed-to-end test. The test here does actually prove that they are removed - otherwise the golden diff would show it, and the JSON patches would look different.